store/migration/10002__add_vcs_github_com.sql

```diff
@@ -6,3 +6,10 @@ ALTER TABLE project_member ADD CONSTRAINT project_member_role_provider_check CHE
 
 ALTER TABLE vcs DROP CONSTRAINT vcs_type_check;
 ALTER TABLE vcs ADD CONSTRAINT vcs_type_check CHECK (type IN ('GITLAB_SELF_HOST', 'GITHUB_COM'));
+
+ALTER TABLE sheet ADD source TEXT NOT NULL CHECK (source IN ('BYTEBASE', 'GITLAB_SELF_HOST', 'GITHUB_COM'));
```

Comment:
Do we need some defaults?

mysql> select * from author1;
+----+------+
| id | name |
+----+------+
|  1 | d    |
+----+------+
1 row in set (0.00 sec)

mysql> ALTER TABLE author1 ADD source TEXT NOT NULL CHECK (source IN ('BYTEBASE', 'GITLAB_SELF_HOST', 'GITHUB_COM'));
ERROR 3819 (HY000): Check constraint 'author1_chk_1' is violated.

Reply:
Literally, I don't think default values should be given. But in order to adapt to the previous version database, it should be added. So do we need to add default values for all added fields in future?

Reply:
Yeah, if we use NOT NULL with CHECK or don't want empty string default, we should add a default.

Reply:
Default value added.